source/platforms/github/GitHubAPI.ts

```diff
@@ -15,6 +16,8 @@ import { RepoMetaData } from "../../dsl/BitBucketServerDSL"
 
 export type APIToken = string
 
+const limit = pLimit(25)
```

Comment:
Note: open to different limits here, this is just a first shot in the dark. Not sure if this'll even fix the issue. 😅

Reply:
It's a good enough start for me